src/nyc_trees/apps/survey/views.py

```diff
@@ -429,6 +428,28 @@ def blockface(request, blockface_id):
     }
 
 
+def blockface(request, blockface_id):
+    blockface = get_object_or_404(Blockface, id=blockface_id)
+    return _blockface_context(blockface)
+
+
+def blockface_near_point(request):
+    p = Point(float(request.GET.get('lng', 0)),
+              float(request.GET.get('lat', 0)),
+              srid=4326)
+
+    # The size of the distance filter was chosen through trial and
+    # error by testing tap precision on a mobile device
+    qs = Blockface.objects.filter(geom__dwithin=(p, 0.0002))\
+                          .distance(p)\
+                          .order_by('distance')
+    blockfaces = qs[:1]  # We only want the closest blockface
+    if blockfaces:
+        return _blockface_context(blockfaces[0])
+    else:
+        raise Http404
```

Comment:
Raising `Http404` leads to a lot of spurious "errors" in the browser console log.  Consider returning `{}` with a status code of 200 instead.